src/server/pps/pretty/pretty.go

```diff
@@ -47,22 +47,14 @@ func PrintJobInfo(w io.Writer, jobInfo *ppsclient.JobInfo) {
 func PrintPipelineHeader(w io.Writer) {
 	// because STATE is a colorful field it has to be at the end of the line,
 	// otherwise the terminal escape characters will trip up the tabwriter
-	fmt.Fprint(w, "NAME\tINPUT\tOUTPUT\tSTATE\t\n")
```

Comment:
I feel like input is a pretty important piece of information, certainly more so than creation time.  Can we just print the set of repos that are used as inputs, regardless of atom/union/cross?  If you wanna do that in another PR that's OK too just make sure to open an issue for it.